### PR TITLE
Add page titles

### PR DESCRIPTION
**Pull request by hharchani**

### CODE REVIEW

File: src/static/scripts/navigation.js [hunk 1]
```diff
@@ -92,6 +93,7 @@ $(function () {
             openLink(pageName);
             updateAltUrls(pageName);
         }
+        document.title = localeBaseTitle;
```
Comment (nisargjhaveri) [inline src/static/scripts/navigation.js:96]: `localeBaseTitle`? :P
Did you mean `localeHomeTitle` or something? :P

Comment (hharchani) [inline src/static/scripts/navigation.js:96]: yeah, like localeBaseUrl :P

Comment (hharchani) [inline src/static/scripts/navigation.js:96]: Do I change it?

**Comment by hharchani** [comment]
This is needed for SEO

**Comment by polybuildr** [comment]
I haven't looked at this in detail, but while I non-AJAX loading of new pages sets the title correctly, using navigation links and AJAX requests doesn't seem to change the title. Was this expected?

**Comment by hharchani** [comment]
Yes, because I don't have the page title in that language. So I set "Felicity IIIT-H" in that language.

**Comment by polybuildr** [comment]
The other part looked okay when tested. Merging.

**Pull request merged**